sljit_src/sljitConfigInternal.h

```diff
@@ -145,7 +144,10 @@ extern "C" {
 #define SLJIT_CONFIG_ARM_64 1
 #elif defined(__thumb2__)
 #define SLJIT_CONFIG_ARM_THUMB2 1
-#elif (defined(__ARM_ARCH) && __ARM_ARCH >= 7) || ((defined(__ARM_ARCH_7__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7R__) || defined(__ARM_ARCH_7S__)) || (defined(__ARM_ARCH_8A__) || defined_(__ARM_ARCH_8R__))
+#elif (defined(__ARM_ARCH) && __ARM_ARCH >= 7) || \
```

Comment:
I am not sure, but I think you mentioned that this macro is always defined. Do we need the rest, or it is just there because of the old code?

Reply:
The new macro should be always defined for compilers that care about ACE (which includes gcc and clang), but sadly not everyone does (ex: tinyc).

Keeping the old macro for backward compatibility (and when properly maintained) helps with that.

Reply:
Ok. It is just a macro.